src/App.tsx

```diff
@@ -1,5 +1,11 @@
+import List from "./List.tsx";
+
 function App() {
-  return <>starting point</>;
+  return (
+    <>
+      <List />
```

Comment:
I couldn't find the Label component in the current file. Could you please check if it's missing or if it needs to be imported from another module?